dotCMS/src/main/java/com/dotmarketing/common/db/DotDatabaseMetaData.java

```diff
@@ -315,6 +315,43 @@ public void executeDropForeignKeyMySql(final Connection conn, final String table
         }
     } // executeDropForeignKeyMySql.
 
+    /**
+     * Returns true if the table already exists
+     * @param connection {@link Connection}
+     * @param tableName  {@link String}
+     * @return Boolean true if exists, otherwise false
+     * @throws SQLException
+     */
+    public boolean existsTable (final Connection connection, final String tableName) throws SQLException {
+
+        boolean exists                        = false;
+        DatabaseMetaData databaseMetaData     = null;
+        ResultSet        resultSet            = null;
+        String           schema               = null;
+        String           table                = tableName;
+
+        try {
+
+            databaseMetaData = connection.getMetaData();
+
+            if (DbConnectionFactory.isOracle()) {
+                table = table.toUpperCase();
+                schema = databaseMetaData.getUserName();
+            }
+
+            resultSet = databaseMetaData.getTables
+                    (connection.getCatalog(), schema, table, null);
+
+            // Iterates over the foreign foreignKey columns
+            exists = resultSet.next();
```

Comment:
Does this sql exception break pgsql's transaction?  I don't think so as it is on a results set and not a query, but I am not sure.  

This method needs a test

Reply:
It does not, it is already tested on an unit test @wezell

Reply:
Well we can do even more test, working on it

Reply:
Done